src/LoginComponent.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class LoginComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.loginUser();
+  }
+
+  loginUser = () => {
+    axios.post('/api/login', {
+      email: this.state.email,
+      password: this.state.password
```

Comment:
Storing JWT in localStorage can be risky due to potential XSS attacks. Consider using HttpOnly cookies or other secure ways to store tokens.